bxml/buildingBlocks/createCall.md

```diff
@@ -0,0 +1,20 @@
+# Create an Outbound Call
```

Comment:
Are we tracking create call in this PR or in the [PR #102](https://github.com/Bandwidth/ap-docs/pull/102)

Reply:
They are different

Reply:
Looks like this file is the same one in the [create call](https://github.com/Bandwidth/ap-docs/pull/102/files#diff-ba54ee7b02e978bb14411df3d46bf659) pull request.